src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -21,7 +24,7 @@ namespace Microsoft.AspNet.OData
     /// </summary>
     /// <typeparam name="TStructuralType">TStructuralType is the type of the instance this delta tracks changes for.</typeparam>
     [NonValidatingParameterBinding]
-    public class Delta<TStructuralType> : TypedDelta, IDelta where TStructuralType : class
+    public class Delta<TStructuralType> : TypedDelta, IDelta, IDeltaSetItem where TStructuralType : class
```

Comment:
>IDeltaSetItem [](start = 62, length = 13)

Since the only things we support in a deltaset are Delta<T> and DeltaDeletedEntity<T> (which derives from Delta<T>), I don't think we need this Interface (at least not in 7.x)

Reply:
Used as a marker interface and contains Deltakind and TransientInstanceAnnotationContainer